### DISCUSSION TITLE
Stop trying to be a social media platform.

### DISCUSSION THREAD

**Discussion by Kurnett** [community/community#55327] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This has been said many times before, but I doubt the changes are going to be reverted unless there's a massive deluge of negative feedback, so...

GitHub is not a social media platform. 

I don't use GitHub to learn about trending new repositories. I don't care what side projects former coworkers from 3+ years ago are working on. I don't want to have more ChatGPT news shoved in my face.

I use GitHub for work. The only thing I want from its home page is quick access to my recently viewed repositories, PRs, and comment threads. Stop trying to turn it into something that it isn't.

**Comment by mzzovko** [comment]
👍

**Comment by moxSedai** [comment]
Just make the main page a list of my repositories in order of recent activity and a panel that shows history.  I go on github to navigate to one of my repositories, not to find new ones or read the github patch notes.

**Comment by wdhdev** [comment]
I personally like the following feed but the for you page and trending is a bit useless.
